subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/declaring_repositories.adoc

```diff
@@ -164,6 +170,9 @@ include::sample[dir="userguide/artifacts/defineRepository/groovy",files="build.g
 include::sample[dir="userguide/artifacts/defineRepository/kotlin",files="build.gradle.kts[tags=maven-like-repo]"]
 ====
 
+[[sub:custom-maven-repo]]
+=== Unusual Maven repositories setup
```

Comment:
:o: Maybe we should turn the title into something a user would want to do? Like "Setting up composite Maven repositories"?

Reply:
Thanks for the suggestion, struggled to find a good title for this

Reply:
By the way, could not find any matching Maven feature this mirrors. Any idea why it was added? Code history does not say much about the why

Reply:
I have no idea